config/jobs/kubernetes/sig-release/release-branch-jobs/1.27.yaml

```diff
@@ -159,7 +159,7 @@ periodics:
       - --allow-dup
       - --bucket=k8s-release-dev
       - --registry=gcr.io/k8s-staging-ci-images
-      - --extra-version-markers=k8s-stable1
+      - --extra-version-markers=k8s-stable2
```

Comment:
This rotation is not supposed to happen now. This should happen only after cutting 1.28.0 and removing 1.24 jobs.

Reply:
Fixed.